src/registry_schemas/schemas/continuation_out.json

```diff
@@ -35,7 +35,8 @@
                     ],
                     "properties": {
                         "country": {
-                            "type": "string"
+                            "type": "string",
+                            "examples": ["CA", "US"]
```

Comment:
Due to duplication, how about if we create a separate schema for foreignJurisdiction and link it here and in the consent schema?

Reply:
I thought about that, but `legalName` is required in continuation out and not needed in consent. what do you think if we move `legalName` from `foreignJurisdiction` to `continuationOut` level @argush3 @severinbeauvais? Then it would be easier to create a separate schema

Reply:
Since `continuationOutDate` is not a property of foreignJuridiction, I think it makes sense for `legalName` to be up a level as well. Another way to look at it is: legal name is not part of the description of the destination "territory" (even though it is tied to that jurisdiction).

On the other hand, our schema for continuationIn.foreignJurisdiction is completely different 🤷‍♂️ 

Does the business care what we do here -- do they have a reason to keep legal name in this object?

Reply:
I'm now thinking we leave this alone -- we have bigger fish to fry :)

Reply:
I started by referring continuationIn schema, thats why legalName is inside foreignJuridiction. But now its completely different. So its the right time to do this change. I will do it right away

Reply:
Well, they're all 3 different. Do whatever you think is right.

Reply:
The new FJ schema looks good to me!

The other 2 filings can be updated later (if ever)... but at least CO is nice and clean.